files/image_config/interfaces/interfaces-config.sh

```diff
@@ -1,6 +1,30 @@
 #!/bin/bash
 
+function wait_networking_service_done() {
+    local -i _WDOG_CNT="1"
+    local -ir _WDOG_MAX="30"
+
+    local -r _TIMEOUT="1s"
+
+    while [[ "${_WDOG_CNT}" -le "${_WDOG_MAX}" ]]; do
+        networking_status="$(systemctl is-active networking 2>&1)"
+
+        if [ "${networking_status}" == active ] || [ "${networking_status}" == inactive ] || [ "${networking_status}" == failed ] ; then
```

Comment:
Please be consistent with '[[' and ']]' as rest of the script.

Reply:
Fixed.